src/App/App.js

```diff
@@ -114,39 +115,33 @@ class App extends Component {
     })
   }
 
-  displayUserRatings(id) {
-    let movieRating;
-    if (!this.state.ratings) {
-      movieRating = "none";
-    } else {
-      const movie = this.state.ratings.find(rating => {
-        return id === rating.movieId;
-      })
-      movieRating = movie ? movie.rating : "none";
-    }
-    return movieRating;
-  }
+  // displayUserRatings(id) {
+  //   let movieRating;
+  //   if (this.state.ratings.length === 0) {
+  //     movieRating = 'none';
+  //   } else {
+  //     const movie = this.state.ratings.find(rating => {
+  //       return id === rating.movieId;
+  //     })
+  //     movieRating = movie ? movie.rating : "Rate Me";
+  //   }
+  //   return movieRating;
+  // }
 
   render() {
     return (
         <main className="App">
-        {this.state.user.id ?
-          <Header 
-            buttonDisplay={this.logOut} 
-            user={this.state.user}
-            buttonText='Log Out'
-            title={`Welcome ${this.state.user.name}`}
-            /> :
           <Header
-            buttonDisplay={this.showLoginPage}
-            buttonText='Log In'
-            title='Welcome to Rancid Tomatillos'
-            />
-        }
+            isUserAuthenticated={this.state.isUserAuthenticated} 
```

Comment:
I was able to get the header displaying dynamically All the logic was moved into the heading component.